crates/orderbook/src/fee.rs

```diff
@@ -338,12 +341,16 @@ impl MinFeeCalculating for MinFeeCalculator {
             } else {
                 let current_fee = self.compute_unsubsidized_min_fee(fee_data).await?;
 
-                if let Err(err) = self
-                    .measurements
-                    .save_fee_measurement(fee_data, internal_valid_until, current_fee)
-                    .await
-                {
-                    tracing::warn!(?err, "error saving fee measurement");
+                if self.store_computed_fees {
+                    if let Err(err) = self
+                        .measurements
+                        .save_fee_measurement(fee_data, internal_valid_until, current_fee)
+                        .await
+                    {
+                        tracing::warn!(?err, "error saving fee measurement");
+                    }
+                } else {
+                    tracing::debug!("skip saving fee measurement");
```

Comment:
Can we set `valid_until` so that it is immediately expired (this way clients don't get the impression that the fee quote they get may still be valid?).

Reply:
Should I use a sentinel value like unix timesamp 0 to indicate that this is intentional?

Reply:
0-sentinel value makes sense.